### PR TITLE
step description update

### PR DESCRIPTION
**Pull request by BanyikAnna**
### Checklist

- [ ] I've read and accepted the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- Requires _MAJOR / MINOR / PATCH_ [version update](https://semver.org/)

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
-title: Set java version
-summary: This step helps you in setting an already installed java version on the virtual machine.
+title: Set Java version
+summary: This Step helps you set an already installed Java version on the virtual machine.
```
Comment (ofalvai) [inline step.yml:2]:
```suggestion
summary: This Step helps you set a pre-installed Java version as default on the build virtual machine.
```

File: step.yml [hunk 2]
```diff
@@ -31,11 +31,11 @@ inputs:
 - set_java_version: "11"
   opts:
     title: Java version to be set globally for the build
-    summary: Select the installed java version you want to use during the build run.
+    summary: Select the installed Java version you want to use during the build run.
```
Comment (ofalvai) [inline step.yml:34]:
```suggestion
    summary: Select which pre-installed Java version you want to use during the build.
```

File: step.yml (hunk 1 above)
Comment (BanyikAnna) [reply step.yml:2]: I love it!

File: step.yml (hunk 2 above)
Comment (BanyikAnna) [reply step.yml:34]: Love it! :)

**Pull request merged**